README.md

```diff
@@ -41,11 +41,13 @@ Note that some form of configuration will be needed so that Nextflow knows how t
 
 4. Start running your own analysis!
 
-   ```bash
-   nextflow run Arcadia-Science/metagenomics --input samplesheet.csv --outdir <OUTDIR> --platform <illumina|nanopore> --sourmash_dbs sourmash_dbs_paths.csv -profile <docker/singularity/conda/institute>
-   ```
+Prior to running the workflow on your samples, you will need to prepare and download databases for `sourmash` and `DIAMOND`. There are a number of [already prepared `sourmash` databases](https://sourmash.readthedocs.io/en/latest/databases.html) available, and you can checkout the [`DIAMOND` documentation](https://github.com/bbuchfink/diamond/wiki) for creating a database compatible with `diamond blastp` used in the workflow.
```

Comment:
for the diamond one, it would be really nice to have an issue we could point to, or a wiki or md file, that has all of the commands that you used to make the diamond database that we use. 

It might also be nice to link to the existing sourmash cover databases here: https://osf.io/6zk3d/